sample/sample.cc

```diff
@@ -194,15 +185,16 @@ const Point<3,double> &p1,const Point<3,double> &p2,const std::array<unsigned in
   s.to_vtk(T,p1,p2,outputName,reader.datatypes,reader.names);
 
 
-  // std::cout << "OK" << std::endl;
+ 
 }
 
-// ./sample 1 infile outfile minx max_x miny maxy minz maxz nx ny nz
-// ./sample 0 infile outfile nx ny nz
+
+
+// ./sample infile outfile minx max_x miny maxy minz maxz nx ny nz
```

Comment:
can you output this as a simple help text if the number of arguments is incorrect? You are also missing the 2 argument case.